### ISSUE TITLE
[postgresql-ha] Using tls with pgpool results in having to roll the pgpool pods everytime the cert renews

### ISSUE THREAD

**Issue by ProNibs** [bitnami/charts#9560]
### Name and Version

bitnami/postgresql-ha:8.6.4

### What steps will reproduce the bug?

Running cert-manager to make pgpool tls certificates means that the tls certificates will get auto-renewed. When the auto-renew occurs, the app will pick up this new tls cert but pgpool will be stuck on the old one.

### Are you using any custom parameters or values?

pgpool:
  tls:
    enabled: true
    

### What is the expected behavior?

Pgpool should be able to dynamically get an updated tls certificate.

### What do you see instead?

The pgpool container uses an initContainer to copy the tls certificate to a new volume and then mounts this new volume.

Instead, just mount the volume directly. If the concern is the owner and the permissions (as that is the main thing the initContainer script does), then use [projected volumes](https://kubernetes.io/docs/concepts/storage/projected-volumes/)

**Comment by fmulero** [comment]
Hi @ProNibs 

Thanks for reporting this issue and information about projected volumes. It looks an interesting and useful feature, unfortunately we don't have enough bandwith to do it in a reasonable time. Would you like to contribute by creating a PR to implement it? The Bitnami team will be happy to review it and provide feedback. [Here](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md) you can find the contributing guidelines.

**Comment by ProNibs** [comment]
Hi @fmulero!

Believe I made a PR, seems like I passed all the initial checks. 
Let me know if you want me to add any more comments or have any feedback on the specific PR. 

I did want to note that postgresql has the same issue with the initContainer, however my workplace strictly uses pgpool for auth and pgpool->postgresql cannot use the tls cert auth. Unsure if that should be a separate PR or not, but it'd be the same process I submitted in the current PR.
